### PR TITLE
Added textAreaRows property to basic input field

### PR DESCRIPTION
**Pull request by skuengneo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.2] - 2023-03-09
```
Comment (neotrow) [inline CHANGELOG.md:10]: Please remove this heading, it will be added automatically by the release workflow

**Review by neotrow — CHANGES_REQUESTED** [review]
Is there also a use case for cols? 

Otherwise looks great apart from the small feedback on the Changelog :-)

File: CHANGELOG.md (hunk 1 above)
Comment (skuengneo) [reply CHANGELOG.md]: Thanks for the input, wasn't sure about this one :P Removed it

**Comment by skuengneo** [comment]
> Is there also a use case for cols?
> 
> Otherwise looks great apart from the small feedback on the Changelog :-)

I don't remember ever using the cols property tbh.

**Pull request merged**